### PR TITLE
B satish sutar patch 6

### PR DESCRIPTION
**Pull request by BSatishSutar**
## Check List (Check all the applicable boxes) 

- [* ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ *] This PR does not contain plagiarized content.
- [ *] The title of my pull request is a short description of the requested changes.

## Note to reviewers

Added few skill sets, result of me upskilling. Please do mind me raising PRs more often, I am upskilling daily and adding more skills, tools etc to my arsenal. Thank you reviewers and maintainers. More power to you..!!

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8771"></a>

### CODE REVIEW

**Review by bharat5604 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 [![Open in GitPod](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree) ![Uptime](https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2FEddieHubCommunity%2Fmonitoring%2Fmaster%2Fapi%2Flink-free-linkfree-io%2Fuptime.json) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![GitHub release (latest by date)](https://img.shields.io/github/v/release/EddieHubCommunity/LinkFree)](https://github.com/EddieHubCommunity/LinkFree/releases) ![GitHub repo size](https://img.shields.io/github/repo-size/EddieHubCommunity/LinkFree)
 
+![2](https://github.com/BSatishSutar/LinkFree/assets/40925459/cd8b1c4d-9d4d-420d-8cc5-7d3e00c200ec)
+
+
```
Comment (SaraJaoude) [inline README.md:5]: I am removing this for the same reasons as I explained on: https://github.com/EddieHubCommunity/LinkFree/pull/8761#issuecomment-1692933613

```suggestion
```

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by SaraJaoude** [comment]
@BSatishSutar I am not sure why you have tried to change the `README.md` again?  I have already explained in https://github.com/EddieHubCommunity/LinkFree/pull/8761 as to why we are not looking for this change, but also our guidelines if you want to suggest any changes other than to your profile.

**Comment by BSatishSutar** [comment]
> @BSatishSutar I am not sure why you have tried to change the `README.md` again? I have already explained in #8761 as to why we are not looking for this change, but also our guidelines if you want to suggest any changes other than to your profile.

I apologize for any confusion. I just found my GitHub inbox today. I won't modify the readme.md file anymore. Thank you for informing me. Also, I'm unsure why I can't see the changes from my recent PR# 8771 , even though Eddie mentioned it was merged.

**Comment by BSatishSutar** [comment]
Attaching the screenshot for your reference:-

image


My concern:- The (.) after Cloud Engineer, Cross-platform DevOps Engineer were replaced by a pipe symbol ( | ) and I added a few skills, which still doesn't reflect on my account.

**Comment by eddiejaoude** [comment]
@BSatishSutar as discussed with you in Discord, you have edited your profile with the forms, and once you start using the forms your json file will be ignored. You can now delete your json file via another pull request and continue to use the forms

**Comment by BSatishSutar** [comment]
> @BSatishSutar as discussed with you in Discord, you have edited your profile with the forms, and once you start using the forms your json file will be ignored. You can now delete your json file via another pull request and continue to use the forms

No Eddie. I have never used forms to edit anything. As a matter of fact i wasn't even aware that we can edit using forms untill yesterday. I would only resort to using the 'forms' option if circumstances necessitate it. And, even If I did use the forms, I still do not see the changes reflecting on my account.

Took this screenshot at 9 am IST today.

image

**Comment by BSatishSutar** [comment]
Oh! So, you're saying that I added repositories using the 'add link' + choice. Does that mean I used forms? I went with that choice to add repositories because I couldn't find any other way to do it. Alright, if my changes won't show up, I'll avoid using the Forms choice in the future. I'm avoiding 'Forms' because I don't want to erase my JSON file at any point. My goal was to learn and create a JSON file to make a pull request for linkfree/biodrop. If I have to delete the JSON file and make all edits using 'Forms', it takes away that excitement, motivation, inner drive within me, and I'm not interested in that.
Anyway, I'm really Grateful to you and your team!

**Pull request merged**